npm-global-without-sudo.md

```diff
@@ -21,13 +21,25 @@ npm config set prefix "${HOME}/.npm-packages"
 Add the following to your `.bashrc`/`.zshrc`:
 
 ```sh
-NPM_PACKAGES="${HOME}/.npm-packages"
-
-export PATH="$PATH:$NPM_PACKAGES/bin"
-
+NPM_PACKAGES="${XDG_DATA_HOME}/npm-packages"
+if ! [ -d "$NPM_PACKAGES" ] && command -v npm >/dev/null 2>&1; then
+  npm config set prefix "$NPM_PACKAGES"
+  mkdir --parents --mode=0700 "$NPM_PACKAGES"
+  npm config set cache "${XDG_CACHE_HOME}/npm"
+  mkdir --parents --mode=0700 "${XDG_CACHE_HOME}/npm"
```

Comment:
npm creates its own directories. No need to do it manually.

Reply:
Thank you for the comment! The latest commit takes it into account.